docs/plugins/hsts.md

```diff
@@ -0,0 +1,20 @@
+_Authors: grenfeldt@google.com_
+
+# HSTS Plugin
+
+HSTS<sup>1</sup> (HTTP Strict Transport Security) tells browsers that they should only access your site using HTTPS and not HTTP. This plugin enforces HSTS by redirecting all HTTP traffic to HTTPS and by setting the `Strict-Transport-Security` header on all outgoing HTTPS traffic.
```

Comment:
"... informs browsers that a website should only be accessed using HTTPS and not HTTP"